e2e/cypress/e2e/security/signUp.cy.ts

```diff
@@ -133,6 +134,16 @@ context('Sign up', () => {
     checkAnonymousUserIdentified();
   });
 
+  it('Signs up and resend email verification', () => {
+    fillAndSubmitSignUpForm(TEST_USERNAME);
+    cy.contains('Thank you for signing up!').should('be.visible');
+
+    cy.contains('Verify your email now');
+
+    gcy('resend-email-button').click();
+    cy.contains('Your verification link has been resent.');
+  });
+
```

Comment:
We should also check that verification link works (if it's not tested somewhere else)